ironfish/src/utils/snapshot.ts

```diff
@@ -2,9 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 export type SnapshotManifest = {
-  block_height: number
+  block_sequence: number
   checksum: string
   file_name: string
   file_size: number
   timestamp: number
+  database_version: number
 }
```

Comment:
This should go in the CLI in export since it's not used by anyone other than the CLI.